toxbat/requirements.py

```diff
@@ -74,6 +58,25 @@ def is_enabled_env(env):
             env.recreate = True
     return config
 
+def all_nested_req_files(requirement_files):
+    """
+    Get all nested req file names for a list of requirements files.
+
+    :param iter requirements_files: list of requirements files to check.
+    :rtype: iter
+    """
+    for reqfile in requirement_files:
+        yield reqfile
+
+        if os.path.isfile(reqfile):
```

Comment:
I assume whole installation won't work if the files do not exist so we shouldn't care about logging (warnings) if the file doesn't exist. I like your pragmatic approach ;)